statsmodels/stats/contrast.py

```diff
@@ -156,11 +156,13 @@ def summary(self, xname=None, alpha=0.05, title=None):
             return summ
         elif hasattr(self, 'fvalue'):
             # TODO: create something nicer for these casee
-            return '<F test: F=%s, p=%s, df_denom=%d, df_num=%d>' % \
-                   (repr(self.fvalue), self.pvalue, self.df_denom, self.df_num)
+            return '<F test: F=%s, p=%s, df_denom=%.3g, df_num=%.3g>' % \
+                   (repr(self.fvalue), self.pvalue, self.df_denom,
+                    self.df_num)
```

Comment:
prettify by putting in parens and removing backslash?

Reply:
changed, given that I had to edit these lines already